tensorboard/data/server/data_compat.rs

```diff
@@ -242,7 +263,11 @@ impl SummaryValue {
                     Some(SCALARS_PLUGIN_NAME) => {
                         md.data_class = pb::DataClass::Scalar.into();
                     }
-                    Some(IMAGES_PLUGIN_NAME) | Some(AUDIO_PLUGIN_NAME) => {
+                    Some(IMAGES_PLUGIN_NAME)
+                    | Some(AUDIO_PLUGIN_NAME)
+                    | Some(GRAPH_RUN_METADATA_PLUGIN_NAME)
+                    | Some(GRAPH_RUN_METADATA_WITH_GRAPH_PLUGIN_NAME)
+                    | Some(GRAPH_KERAS_MODEL_PLUGIN_NAME) => {
```

Comment:
Future plan question: are we going to trust the `data_class` property on the proto in the future? I believe the graph_run* and graph_keras* data are all annotated correctly with the data_class.

Reply:
We already trust the data class: the first clause of this `match` is

```rust
        match (md, &*self.0) {
            // Any summary metadata that sets its own data class is expected to already be in the right
            // form.
            (Some(md), _) if md.data_class != i32::from(pb::DataClass::Unknown) => Box::new(md),
```

But I’m not sure if the graph data does set data classes? The data
written by `:graphs_demo` doesn’t seem to have them, and I don’t see
data classes in [the writing code][w]. In any case, data classes were
introduced more recently than the `graph_*` writing code, so old data
must not have them, I think.

So eventually I would probably like plugins to write summaries with data
class explicitly set, but I expect to maintain this compatibility
indefinitely, and that seems okay to me. WDYT?

[w]: https://github.com/tensorflow/tensorflow/blob/d7485ac5c3b7c76100fb5854af50b76322d8ec58/tensorflow/python/ops/summary_ops_v2.py#L1156-L1161